config/swagger.yaml

```diff
@@ -920,18 +920,30 @@ paths:
       security: []
 
 definitions:
+  EncodedHash:
+    type: string
+  Pow:
+    type: array
+    items:
+      type: integer
+      format: int32
+    minItems: 42
+    maxItems: 42
+  Uri:
+    type: string
+    ## format: uri
```

Comment:
I understand this can be safely uncommented (documentation [here](https://github.com/OAI/OpenAPI-Specification/blob/ba49b592c13594d06db4ea99fe4cd795b3d83ace/versions/2.0.md#data-types)).

Reply:
The Swagger specification tells us that uri is a valid format, but it is not accepted by the tool chain!